legacy/README.md

```
@@ -1,37 +1,414 @@
+   # Redis & Background Jobs
+   REDISHOST=127.0.0.1
+   REDISPORT=6379
+   BROKER_URL=redis://127.0.0.1:6379/0
+   CELERY_QUEUE_NAME=dev
+
+   # Project & Repo Management
+   PROJECT_PATH=projects
+   ```
+
+   > **`CHAT_MODEL`** and **`INFERENCE_MODEL`** are used for agent reasoning and knowledge graph generation respectively. Model names follow the `provider/model_name` format as expected by [LiteLLM](https://docs.litellm.ai/docs/providers).
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)